### DISCUSSION TITLE
Feature Request: Ability to Change in Settings Redirect Route after Login and Homepage

### DISCUSSION THREAD

**Discussion by matbrgz** [outline/outline#1434] [General]
You could choose stay using home or each collection redirection as default route.

**Comment by tommoor** [comment]
I think this speaks to the home screen being not useful enough, and the _real_ ask here is to give Home more functionality?

**Comment by matbrgz** [comment]
Ability to add a text in home could be a good start. Then I think in bring metrics on home.
